stdlib/public/core/OutputStream.swift

```diff
@@ -386,13 +387,17 @@ internal func _print_unlocked<T, TargetStream : TextOutputStream>(
 ///
 /// This function is forbidden from being inlined because when building the
 /// standard library inlining makes us drop the special semantics.
+@_inlineable
```

Comment:
I see. In practice does specializing here make a difference? Perhaps just having some pre-specializations is sufficient.